src/mlpack/core/util/arma_traits.hpp

```diff
@@ -301,15 +311,23 @@ struct IsBaseMatType<arma::SpRow<eT>>
 template<typename T>
```

Comment:
We should add a comment indicating this is true for *any* (non-field) Armadillo type, be it sparse, dense, or a cube.